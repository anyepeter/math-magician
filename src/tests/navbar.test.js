import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navigation from '../pages/navbar';

describe('navbar render links', () => {
  it('check if it rendering or not', () => {
    const { container } = render(
      <BrowserRouter>
        <Navigation />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
