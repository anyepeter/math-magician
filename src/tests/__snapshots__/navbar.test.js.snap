// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar render links check if it rendering or not 1`] = `
<div>
  <div
    class="nav-bar"
  >
    <div
      class="logo"
    >
      <h1
        class="magicians"
      >
        Math Magicians
      </h1>
    </div>
    <nav
      class="nav"
    >
      <a
        class="links"
        href="/"
      >
        Home
      </a>
      |
      <a
        class="links"
        href="/calculator"
      >
        Calculator
      </a>
      |
      <a
        class="links"
        href="/Quotes"
      >
        Quotes
      </a>
    </nav>
  </div>
</div>
`;
